bundle-workflow/Jenkinsfile

```diff
@@ -46,6 +46,7 @@ pipeline {
                     post() {
                         always {
                             cleanWs disableDeferredWipeout: true, deleteDirs: true
+                            sh "rm -rf ~/.m2/"
```

Comment:
Can we have this in a `clean_up` function in this script so we can add all the common stuff that needs to be removed in one place?